src/Modules/GraphEngine.DynamicCluster/Trinity.Azure.Storage.Test/Trinity.Azure.Storage.Test.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphEngine.Core" Version="$(DOTNET_PACKAGE_VERSION)" />
+    <PackageReference Include="GraphEngine.Core" Version="2.0.9608" />
```

Comment:
please rollback the changes specifying a fixed version. $(DOTNET_PACKAGE_VERSION) is introduced to update the version automatically.

Reply:
@yatli Yep - I already caught it; local compilations and builds work against the build process as VS is making these changes to the project configuration files.

Reply:
Changed applied